src/app/store/collectibles/reducer.ts

```diff
@@ -1,20 +1,51 @@
 import { createReducer } from '@reduxjs/toolkit';
+import { persistReducer } from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
 
 import { createEntity } from 'lib/store';
 
 import { loadCollectiblesDetailsActions } from './actions';
 import { collectiblesInitialState, CollectiblesState } from './state';
 
-export const collectiblesReducer = createReducer<CollectiblesState>(collectiblesInitialState, builder => {
+/** In seconds // TTL = Time To Live */
+const ADULT_FLAG_TTL = 3 * 60 * 60;
+
+const collectiblesReducer = createReducer<CollectiblesState>(collectiblesInitialState, builder => {
   builder.addCase(loadCollectiblesDetailsActions.submit, state => {
     state.details.isLoading = true;
   });
-  builder.addCase(loadCollectiblesDetailsActions.success, (state, { payload }) => ({
-    ...state,
-    details: createEntity({ ...state.details.data, ...payload })
-  }));
+  builder.addCase(loadCollectiblesDetailsActions.success, (state, { payload }) => {
+    const adultFlags = { ...state.adultFlags };
+    const now = Math.round(Date.now() / 1_000);
```

Comment:
Sorry, using functions `Date.now()` in reducers is discouraged. Please consider moving this logic somewhere else

Reply:
agree with Kesha, we need to avoid Date.Now() inside reducers
https://redux.js.org/tutorials/fundamentals/part-3-state-actions-reducers#rules-of-reducers